perftests/apiserver/loadtest/apiserver_benchmarker/main.go

```
@@ -144,11 +145,12 @@ func validateArgs(a *loadtest.Args, startAsServer bool, port string) bool {
 		err = true
```

Can we return the particular err that isn't valid and bubble it up to argParse()? As it currently exists, we won't even get to the end of validateArgs if one isn't valid, as log.Fatal calls an os.Exit(1)

---

changed fatalf to errorf. performing all edits before returning.